K2Field.K2NE.ServiceBroker/ServiceObjects/EmailTemplate/Template.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Text;
+using System.Text.RegularExpressions;
+using SourceCode.SmartObjects.Client;
+
+namespace K2Field.K2NE.ServiceBroker.ServiceObjects.EmailTemplate
+{
+    /// <summary>
+    /// The Class is used to work with EmailTemplates. 
```

Comment:
It but it also does part of the translation and so on. The responsibility of the class is not supper clear to me. That results also in code where this class is used that it's not 100% clear how it works. the constructor does the actual work etc.